tests/foreman/rhai/test_rhai.py

```diff
@@ -98,3 +99,25 @@ def test_rhai_client_setup(self):
             Navigator(self.browser).go_to_insights_systems()
             result = self.rhai.view_registered_systems()
             self.assertIn("1", result, 'Registered clients are not listed')
+
+    def test_org_selection_for_rhai(self):
+        """@Test:- Verify that user attempting to access RHAI is directed to
+        select an Organization if there is no organization selected
+
+        @Feature: In order to use Access Insights, user must select an
+        organization
+
+        @Assert: 'Organization Selection Required' message must be displayed if
+        the user tries to view Access Insights overview without selecting an
+        org
+        """
+
+        with Session(self.browser) as session:
+            # Given that the user does not specify any Organization
+            session.nav.go_to_select_org("Any Organization")
+            session.nav.go_to_insights_overview()
+
+            # 'Organization Selection Required' message must be present
+            result = session.nav.wait_until_element(
+                locators['insights.org_selection_msg']).text
+            self.assertIn("Organization Selection Required", result)
```

Comment:
Yes, will be run only in English for now. I shall take care of the language part sometime later, not an immediate concern though.